src/emu/natkeyboard.h

```diff
@@ -33,6 +33,12 @@ class natural_keyboard
 	DISABLE_COPYING(natural_keyboard);
 
 public:
+	enum
+	{
+		SHIFT_COUNT = 2,
+		SHIFT_STATES = 1 << SHIFT_COUNT
+	};
```

Comment:
This really shouldn't be public - nothing else should be depending on it.  Also, you can define `SHIFT_COUNT` here in terms of the `UCHAR_SHIFT_` macros given you know you'll have `emu.h` before `natkeyboard.h` due to the PCH nonsense.